src/connect.ts

```diff
@@ -88,9 +90,20 @@ export function createConnect<StateType = any, GettersType = any>(
         name = nameOrComponent
       }
 
+      if (typeof gettersToProps == 'function') {
+        var evaluatedGettersToProps = gettersToProps(mapGetters);
```

Comment:
Please avoid using `var`. You can use ternary operator and `const` here.

```ts
const evaluatedGettersToProps = typeof gettersToProps === 'function'
  ? gettersToProps(mapGetters)
  : mapGetters(gettersToProps)
```

Same for `stateToProps`.